packages/flutter_tools/test/general.shard/android/android_device_discovery_test.dart

```diff
@@ -17,7 +17,7 @@ import '../../src/fake_process_manager.dart';
 import '../../src/testbed.dart';
 
 void main() {
-  testWithoutContext('AndroidDevices returns empty device list on null adb', () async {
+  testWithoutContext('AndroidDevices returns empty device list and diagnostics on null adb', () async {
```

Comment:
unskipped test that failed without addition of "just in case" null checks above.